src/Microsoft.AspNetCore.Http/Extensions/HttpRequestRewindUtility.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Http.Internal;
+
+namespace Microsoft.AspNetCore.Http.Extensions
+{
+    public static class HttpRequestRewindUtility
+    {
+        /// <summary>
+        /// Ensure the <paramref name="request"/> <see cref="HttpRequest.Body"/> can be read multiple times.
+        /// </summary>
+        /// <param name="request">The <see cref="HttpRequest"/> to prepare.</param>
+        /// <returns>The updated <paramref name="request"/>.</returns>
+        public static HttpRequest EnableRewind(HttpRequest request)
+        {
+            return BufferingHelper.EnableRewind(request);
+        }
+
+        /// <summary>
+        /// Ensure the <paramref name="request"/> <see cref="HttpRequest.Body"/> can be read multiple times.
+        /// </summary>
+        /// <param name="request">The <see cref="HttpRequest"/> to prepare.</param>
+        /// <param name="bufferThreshold">
+        /// The maximum size in bytes of the in-memory <see cref="System.Buffers.ArrayPool{Byte}"/> used to buffer the
+        /// stream. Larger request bodies are written to disk.
```

Comment:
We write to disk in all the cases of these methods. We should have a similar remark but mentioning the default threshold (and its value) in the method on line no. 15